api/app/Constants/EventConstants.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Constants;
+
+class EventConstants
+{
+    public const DEFAULT_STATUS = 'scheduled';
```

Comment:
The name of the constant should represent the value (`SCHEDULED` here). You can check the JsonResponse class source for the example.